src/codegen/metadata/script_info.py

```diff
@@ -3,6 +3,10 @@
         {
             "relativeOutputPath": "_task_modules\\channels\\ai_channel.py",
             "templateFile": "_task_modules\\channels\\ai_channel.py.mako",
+        },
+        {
+            "relativeOutputPath": "_task_modules\\channels\\ai_channel_collection.py",
```

Comment:
This path should also be changes to point to the _task_modules directory. The output path should match the existing location of the ai_channel_collection.py

Reply:
Updated.